### ISSUE TITLE
EXPORT TERMINAL COMMAND - free(): double free detected in tcache 2

### ISSUE THREAD

**Issue by goodevile** [ultralytics/ultralytics#21176]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Export

### Bug

Nothing but the export command in the terminal.
using WSL2 (Windows Subsystem for Linux)

`yolo export model=/home/xxxx/models/best.pt format=onnx`

```
free(): double free detected in tcache 2
Aborted (core dumped)
```

Ultralytics 8.3.159 🚀 Python-3.12.3 torch-2.7.0+cu126 CPU (AMD Ryzen 7 5700X3D 8-Core Processor)
YOLO11n summary (fused): 100 layers, 2,582,737 parameters, 0 gradients, 6.3 GFLOPs

PyTorch: starting from '/home/xxxxxx/models/best.pt' with input shape (1, 3, 352, 352) BCHW and output shape(s) (1, 7, 2541) (5.2 MB)

### Environment

OS                  Linux-6.6.87.1-microsoft-standard-WSL2-x86_64-with-glibc2.39
Environment         Linux
Python              3.12.3
Install             pip
Path                /home/xxxxxx/yolo/lib/python3.12/site-packages/ultralytics
RAM                 54.92 GB
Disk                173.7/1006.9 GB
CPU                 AMD Ryzen 7 5700X3D 8-Core Processor
CPU count           16
GPU                 NVIDIA GeForce RTX 3090, 24576MiB
GPU count           1
CUDA                12.6

numpy               ✅ 2.2.5>=1.23.0
matplotlib          ✅ 3.10.3>=3.3.0
opencv-python       ✅ 4.11.0.86>=4.6.0
pillow              ✅ 11.2.1>=7.1.2
pyyaml              ✅ 6.0.2>=5.3.1
requests            ✅ 2.32.3>=2.23.0
scipy               ✅ 1.15.3>=1.4.1
torch               ✅ 2.7.0>=1.8.0
torch               ✅ 2.7.0!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         ✅ 0.22.0>=0.9.0
tqdm                ✅ 4.67.1>=4.64.0
psutil              ✅ 7.0.0
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 2.2.3>=1.1.4
ultralytics-thop    ✅ 2.0.14>=2.0.0

### Minimal Reproducible Example

activate the venv then:` yolo export model=/home/xxxx/models/best.pt format=onnx`

### Additional

Package                            Version
---------------------------------- -----------
alembic                            1.15.2
annotated-types                    0.7.0
anyio                              4.9.0
asttokens                          3.0.0
blinker                            1.9.0
cachetools                         5.5.2
certifi                            2025.4.26
charset-normalizer                 3.4.2
click                              8.2.0
cloudpickle                        3.1.1
colorlog                           6.9.0
comm                               0.2.2
contourpy                          1.3.2
cycler                             0.12.1
databricks-sdk                     0.53.0
debugpy                            1.8.14
decorator                          5.2.1
Deprecated                         1.2.18
docker                             7.1.0
executing                          2.2.0
fastapi                            0.115.12
filelock                           3.18.0
Flask                              3.1.1
fonttools                          4.58.0
fsspec                             2025.3.2
gitdb                              4.0.12
GitPython                          3.1.44
google-auth                        2.40.1
graphene                           3.4.3
graphql-core                       3.2.6
graphql-relay                      3.2.0
greenlet                           3.2.2
gunicorn                           23.0.0
h11                                0.16.0
idna                               3.10
importlib_metadata                 8.6.1
ipykernel                          6.29.5
ipython                            9.2.0
ipython_pygments_lexers            1.1.1
itsdangerous                       2.2.0
jedi                               0.19.2
Jinja2                             3.1.6
joblib                             1.5.0
jupyter_client                     8.6.3
jupyter_core                       5.7.2
kiwisolver                         1.4.8
Mako                               1.3.10
Markdown                           3.8
MarkupSafe                         3.0.2
matplotlib                         3.10.3
matplotlib-inline                  0.1.7
mlflow                             2.22.0
mlflow-skinny                      2.22.0
mpmath                             1.3.0
nest-asyncio                       1.6.0
networkx                           3.4.2
numpy                              2.2.5
nvidia-cublas-cu12                 12.6.4.1
nvidia-cuda-cupti-cu12             12.6.80
nvidia-cuda-nvrtc-cu12             12.6.77
nvidia-cuda-runtime-cu12           12.6.77
nvidia-cudnn-cu12                  9.5.1.17
nvidia-cufft-cu12                  11.3.0.4
nvidia-cufile-cu12                 1.11.1.6
nvidia-curand-cu12                 10.3.7.77
nvidia-cusolver-cu12               11.7.1.2
nvidia-cusparse-cu12               12.5.4.2
nvidia-cusparselt-cu12             0.6.3
nvidia-nccl-cu12                   2.26.2
nvidia-nvjitlink-cu12              12.6.85
nvidia-nvtx-cu12                   12.6.77
opencv-python                      4.11.0.86
opentelemetry-api                  1.33.0
opentelemetry-sdk                  1.33.0
opentelemetry-semantic-conventions 0.54b0
optuna                             4.3.0
packaging                          24.2
pandas                             2.2.3
parso                              0.8.4
pexpect                            4.9.0
pillow                             11.2.1
pip                                24.0
platformdirs                       4.3.8
prompt_toolkit                     3.0.51
protobuf                           6.31.0
psutil                             7.0.0
ptyprocess                         0.7.0
pure_eval                          0.2.3
py-cpuinfo                         9.0.0
pyarrow                            19.0.1
pyasn1                             0.6.1
pyasn1_modules                     0.4.2
pydantic                           2.11.4
pydantic_core                      2.33.2
Pygments                           2.19.1
pyparsing                          3.2.3
python-dateutil                    2.9.0.post0
pytz                               2025.2
PyYAML                             6.0.2
pyzmq                              26.4.0
requests                           2.32.3
rsa                                4.9.1
scikit-learn                       1.6.1
scipy                              1.15.3
seaborn                            0.13.2
setuptools                         80.7.1
six                                1.17.0
smmap                              5.0.2
sniffio                            1.3.1
SQLAlchemy                         2.0.41
sqlparse                           0.5.3
stack-data                         0.6.3
starlette                          0.46.2
sympy                              1.14.0
threadpoolctl                      3.6.0
torch                              2.7.0
torchvision                        0.22.0
tornado                            6.5
tqdm                               4.67.1
traitlets                          5.14.3
triton                             3.3.0
typing_extensions                  4.13.2
typing-inspection                  0.4.0
tzdata                             2025.2
ultralytics                        8.3.159
ultralytics-thop                   2.0.14
urllib3                            2.4.0
uvicorn                            0.34.2
wcwidth                            0.2.13
Werkzeug                           3.1.3
wrapt                              1.17.2
zipp                               3.21.0

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @goodevile, thank you for reporting this issue and providing detailed system information! 🚀 This is an automated response to help you get started. An Ultralytics engineer will assist you here soon.

We recommend reviewing the [Docs](https://docs.ultralytics.com/) for additional guidance, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many common questions and issues may already be addressed there.

Since this appears to be a 🐛 bug report, thank you for including a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/)—this helps us a lot in debugging!

If you have any additional logs, error tracebacks, or further details about the export command and the model you used, please feel free to add them here.

Join the Ultralytics community for support and discussion:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth topics: [Discourse](https://community.ultralytics.com/)
- Community sharing: [Subreddit](https://www.reddit.com/r/Ultralytics/)

## Upgrade

Please make sure you are using the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to confirm the issue persists:

```bash
pip install -U ultralytics
```

## Environments

YOLO can run in any of these up-to-date, verified environments (with all dependencies, including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If the badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you again for your report and patience! An Ultralytics engineer will follow up with you here.

**Comment by Y-T-G** [comment]
Duplicate of https://github.com/ultralytics/ultralytics/issues/21124
